core/src/main/scala/akka/kafka/javadsl/Producer.scala

```diff
@@ -12,7 +12,7 @@ import akka.kafka.{ConsumerMessage, ProducerSettings}
 import akka.kafka.ProducerMessage._
 import akka.kafka.scaladsl
 import akka.stream.javadsl.{Flow, Sink}
-import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
+import org.apache.kafka.clients.producer.{Producer => KafkaProducer, ProducerRecord}
```

Comment:
Indeed here `Producer` would clash, but it'd still be nice to avoid confusion with the 'real' `KafkaProducer` - perhaps use `KProducer`?

Reply:
changed this in 564a2ad